index.js

```diff
@@ -42,7 +42,7 @@ const getWslDrivesMountPoint = (() => {
 		}
 
 		const configContent = await fs.readFile(configFilePath, {encoding: 'utf8'});
-		const configMountPoint = /root\s*=\s*(?<mountPoint>.*)/g.exec(configContent);
+		const configMountPoint = /(?<!#.*)root\s*=\s*(?<mountPoint>.*)/g.exec(configContent);
```

Comment:
The regex results in an error: https://regex101.com/r/4pJXHS/1

Reply:
I'm not sure I see the error, has the regex101 test string changed? I just see `#root` and `root` below it which doesn't match, but that's no different to the original regex because it expects `=` after root. If I change the second line to `root=/` it finds the match group. 

I've tested this change locally with a `#root` comment directly above the actual mount point setting and it worked as expected.

Reply:
Ok. It turns out I was looking at it with Safari and Safari doesn't support negative lookbehind.